flask_restless/helpers.py

```diff
@@ -622,3 +640,36 @@ def __call__(self, model, _apimanager=None, **kw):
 #:     <class 'mymodels.Person'>
 #:
 model_for = ModelFinder()
+
+#: Returns the primary key to be used for the given model, as specified
```

Comment:
OK, have fixed a silly mistake in the documentation in f2bd109 and added an extra example (I think this is what you were after here?) in f6cf17d.